enable/drag.py

```diff
@@ -39,7 +39,7 @@ def init(self, components, bounds_rect, drag_root, drag_bounds_rect,
         self.window = components[0].window
         self.drag_over = []
 
-        self.on_trait_change(self.drag_done, drag_event)
+        self.observe(self.drag_done, drag_event)
```

Comment:
How do we know `drag_event` is compatible with `observe`?

Reply:
Looking into this more I am quite confused... 

So the this `DragHandler` class is defining an `init` not `__init__` method.  However, it subclasses `Interactor`, which subclasses `HasTraits`.  Neither of these subclasses define/call an `init` method.  This is something that is defined for traitsUI editors to do additional initialization.

I put a print statement in the method and ran the test suite only to see it was never called.  This obviously doesn't guarantee anything but it raised some red flags about this class.  

searching for `DragHandler` in the enable and chaco codebases yielded no results.  I then searched on github and found no results (although my view of the full Enthought github is limited).

I don't think the code in this file would have worked correctly previously as I don't see when `init` could ever have been called, and then any other methods accessing `self._____` would fail as well... I also cant see the class used anywhere.  

@jwiggins should we consider removing this code?  It is very possible I am missing something and we want to keep it, but currently I am pretty confused

Reply:
likewise for `DragResizeHandler` in `enable/drag_resize.py`

Reply:
did a little more github digging, and outside of copyright headers, and style fixes the `enable/drag.py` file effectively hasn't been touched since it was written... which was in 2003

Reply:
#685 explicitly names this module.

In the context of this PR what do you want to do? Leave it untouched, or edit it so that it doesn't show up when looking for old notification patterns?

Reply:
Ah that makes sense, I hadn't seen that on 685 thanks!

I am happy to just leave it untouched here